PreFreeSurfer/scripts/BiasFieldCorrection_T1wOnly.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -e
+source ${HCPPIPEDIR}/global/scripts/debug.shlib # Debugging functions
+
+# Requirements for this script
+#  installed versions of: FSL (version 5.0.6), caret7 (a.k.a. Connectome Workbench) (version 1.0)
+#  environment: FSLDIR  CARET7DIR
```

Comment:
Is Workbench (CARET7DIR) actually needed for anything in this script?

Reply:
Implemented now.